### PR TITLE
Proxies arg was being passed a unicode object when a dict is required. U...

### PR DESCRIPTION
**Pull request by jaloren**
I am on a corporate network that forces me to go through a HTTP proxy. I discovered that when I set the argument for the proxies in a test case, the connection would timeout. After some debugging, I discovered that while the proxies argument was being set, the value was not of the right type. It was a string instead of a dict. This effectively disabled the proxy argument for the session object.

Modified the library so that the proxies argument is converted to a dict so that the session object uses that value correctly if set.

### CODE REVIEW

File: src/RequestsLibrary/RequestsKeywords.py [hunk 1]
```diff
@@ -57,9 +58,12 @@ def _create_session(self, alias, url, headers, cookies, auth,
         s = session = requests.Session()
         s.headers.update(headers)
         s.auth = auth if auth else s.auth
-        s.proxies = proxies if proxies else  s.proxies
-
         s.verify = self.builtin.convert_to_boolean(verify)
+        if proxies:
+            s.proxies = ast.literal_eval(proxies)
```
Comment (bulkan) [inline src/RequestsLibrary/RequestsKeywords.py]: Thanks for the PR. Instead of this line could you not use `Create Dictionary` in the test case file ?

Comment (jaloren) [inline src/RequestsLibrary/RequestsKeywords.py]: Actually, yes that would work. 

I guess the question is philosophical -- add intermediate data type conversions in the test case itself or expect the library to handle it? I personally favor the latter because it makes for a cleaner test case.

That said if the current behavior is preserved, could we at least add a documentation example that demonstrates this? I had a heck of a time tracking down the problem -- it was just failing silently instead of throwing a type error.

Comment (bulkan) [inline src/RequestsLibrary/RequestsKeywords.py]: Well; the [Zen of Python](https://www.python.org/dev/peps/pep-0020/) writes;

>   Explicit is better than implicit.
 
That was the philosophy behind the approach I took to writing test cases as well.

Comment (jaloren) [inline src/RequestsLibrary/RequestsKeywords.py]: sure...will do a separate PR for that. This one can be be closed.
